src/extras/MovieClip.js

```diff
@@ -123,7 +151,12 @@ Object.defineProperties(MovieClip.prototype, {
     currentFrame: {
         get: function ()
         {
-            return Math.floor(this._currentTime) % this._textures.length;
+            var currentFrame = Math.floor(this._currentTime) % this._textures.length;
+            if (currentFrame < 0)
```

Comment:
I'm confused by this, what is it here for? Looks like it is making it loop, but starting at the end of the list?